security/cert_manager_operator/index.adoc

```diff
@@ -20,9 +20,15 @@ include::modules/cert-manager-request-methods.adoc[leveloffset=+1]
 //Supported versions
 include::modules/cert-manager-supported-versions.adoc[leveloffset=+1]
 
+//FIPS compliant support
+include::modules/cert-manager-fips-support.adoc[leveloffset=+1]
+
+include::modules/cert-manager-fips-procedure.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 [id="cert-manager-operator-about_additional-resources"]
 == Additional resources
 
 * link:https://cert-manager.io/docs/[cert-manager project documentation]
+
+* link:https://access.redhat.com/articles/5059881[OpenShift security guide book]
```

Comment:
As discussed, consider double-checking if the security guide book is publicly accessible and if yes, if it's a good fit to share with customers/partners externally for their reference.